src/objects.c

```diff
@@ -3186,3 +3938,51 @@ CK_RV p11prov_merge_pub_attrs_into_priv(P11PROV_OBJ *pub_key,
     P11PROV_raise(priv_key->ctx, ret, "Failed attr copy");
     return CKR_GENERAL_ERROR;
 }
+
+/* creates an empty (no public point) Public EC Key, (OpenSSL paramgen
+ * function), that will later be filled in with a public EC key obtained
+ * by a peer */
```

Comment:
This could be used for other reasons too, so it is somewhat vague intentionally.
(our use case is driven by ECDH, but the code at this point does not actually know that)